chore: include IDEs' garbage files in the gitignore

413x1nkp opened the pull request:
while working on this repo in vscode/vscodium, a folder with vscode's workspace settings will appear, called `.vscode`; this is also the case for Intellij's PyCharm and Xcode. since those settings provide nothing meaningful to end-users and are practically just another pain to deal with for the devs, it's better to just add those to `.gitignore`

each IDEs' config dir entries in gitignore are separated in their own commits, in case the decision would be to not add some entry to the upstream gitignore file

fumiama commented:
This repo is not main and I have transferred this PR to main repo. You should PR to there https://github.com/RVC-Project/Retrieval-based-Voice-Conversion-WebUI next time🤝

413x1nkp committed:
chore: add gitignore entry for .vscode garbage files

while working on this repo in vscode/vscodium, a folder with vscode's workspace settings will appear, called `.vscode`. since those settings provide nothing meaningful to end-users and are practically just another pain to deal with, it's better to just add it to .gitignore

413x1nkp committed:
chore: add IDEA's workspace config dir to the gitignore

413x1nkp committed:
chore: add Xcode's workspace config dir to the gitignore

fumiama reviewed (approved):
Thank you.
